Neos.Utility.ObjectHandling/Classes/ObjectAccess.php

```diff
@@ -114,6 +115,9 @@ protected static function getPropertyInternal($subject, string $propertyName, bo
         $className = TypeHandling::getTypeForValue($subject);
 
         if ($forceDirectAccess === true) {
+            if ($subject instanceof ProxyInterface) {
```

Comment:
Just a question: Isn't this something Neos::TypeHandling should do as well? It does exactly the same thing for Doctrine-Proxies already, which is no "Flow" class, but it does not for our own Proxies? Feels weird.

Reply:
I agree, this should probably be fixed here https://github.com/neos/flow-development-collection/blob/475c71bc569f3900e15eb0a917ce49da819ac8ed/Neos.Utility.ObjectHandling/Classes/TypeHandling.php#L172-L184

Reply:
Valid point